frontend/src/pages/Admin/Flags.js

```diff
@@ -47,6 +56,28 @@ export default function Flags() {
                 {' '}
                 &nbsp; &nbsp;
                 <Link to={`/admin/users?flag=${feature}`} aria-label={`view users with the ${feature} feature flag`}>View active users</Link>
+                {' '}
+                &nbsp; &nbsp;
+                {feature === 'anv_statistics' && (
```

Comment:
any reason not to just have this button for every flag?

Reply:
The plan was to provide a "low-cost" solution that would allow Patrice to better synchronize sending emails to users and turning on the anv statistics flag. There was some concern that if we provided buttons for other flags, admins might "play" with them and accidentally lose a subset of users who had flags enabled. Perhaps we shouldn't be concerned about that, in which case there aren't any other reasons. We may still remove feature flags once they are no longer required.